chrome-extension/background.js

```diff
@@ -162,10 +195,14 @@ chrome.downloads.onCreated.addListener(function(downloadItem) {
     chrome.downloads.erase({ id: downloadItem.id }); // Erase the download from list
 
     clearMessage();
+    chrome.cookies.getAll({'url' : extractRootURL(info.pageUrl)}, parseCookies);
```

Comment:
@putrasattvika `info.pageUrl` is not available at this line. I believe it should be `downloadItem['url']`, `downloadItem['finalUrl']` or `downloadItem['referrer']`.